java/arcs/core/storage/driver/Database.kt

```diff
@@ -92,6 +92,12 @@ object DatabaseDriverProvider : DriverProvider {
         ).register()
     }
 
+    override suspend fun removeAllEntities() { manager.removeAllEntities() }
```

Comment:
Either use function-expression notation, or new lines between braces and the function body.
```suggestion
    override suspend fun removeAllEntities() { 
        manager.removeAllEntities() 
    }
```